src/components/structures/MatrixChat.js

```diff
@@ -1120,7 +1132,6 @@ module.exports = React.createClass({
                     sessionId={this.state.register_session_id}
                     idSid={this.state.register_id_sid}
                     email={this.props.startingFragmentQueryParams.email}
-                    referrer={this.props.startingFragmentQueryParams.referrer}
```

Comment:
ick. I wonder if the `_makeRegistrationUrl` wrapper to add the referrer belongs in `Registration` then. I'm bored of this PR though, let's just get it shipped